### PR TITLE
Adding styles.css for the home page

### PR DESCRIPTION
**Pull request by yellow-tulips**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,409 @@
+@charset "UTF-8";
```
Comment (Taraque-A) [inline styles.css:1]: Great works!

**Review by Taraque-A — APPROVED** [review]

**Review by tasnhubaislam — APPROVED** [review]
Looks great!

**Review by farzanaisrat — APPROVED** [review]
looks good!

**Review by joerossi215 — APPROVED** [review]
Looks good to go!!

**Pull request merged**